images/renderer/renderer.py

```diff
@@ -28,9 +30,27 @@ def render_ipynb(full_path, format):
         html, res = exporter.from_file(file_handle)
     return Response(html, mimetype='text/html')
 
+def render_md(full_path, format):
+    """
+    Render a given .md file
+    """
+    md = markdown.markdownFromFile(filename)
+    rmd = md.convert()
+    return rmd
+
+def render_rst(full_path, format):
+    """
+    Render a given .rst file
+    """
+    docutils.core.publish_file(source_path=full_path, destination path="output.html", writer_name="html")
```

Comment:
typo: destination_path.

Moreover, is this writing to the disk? Would it be possible to access the resulting HTML directly from memory?